client/dashboard/app/i18n/index.tsx

```diff
@@ -0,0 +1,26 @@
+import defaultCalypsoI18n from 'i18n-calypso';
+import { useEffect } from 'react';
+import { setupLocale } from 'calypso/boot/locale';
+import CalypsoI18nProvider from 'calypso/components/calypso-i18n-provider';
+import switchLocale from 'calypso/lib/i18n-utils/switch-locale';
+import { useAuth } from '../auth';
+
+export function I18nProvider( { children }: { children: React.ReactNode } ) {
+	const { user } = useAuth();
```

Comment:
I checked out this locally. I found that the new language I set in `/me` isn't reflected here unless I comment this line out 🥲 are you experiencing the same?

https://github.com/Automattic/wp-calypso/blob/fa2bc0ffd8bf93c2511c1d1507d7e9dc0350bdda/client/dashboard/app/auth/index.tsx#L29

Reply:
We probably need to invalidate React Query caches for anything that depends on `/me` information when we know that we've changed that information (i.e. updated the language in the interface).

https://tanstack.com/query/latest/docs/framework/react/guides/query-invalidation